src/ucp/wireup/select.c

```diff
@@ -720,7 +739,18 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_add_lane_desc(
         }
     }
 
-    if (select_ctx->num_lanes >= UCP_MAX_LANES) {
+    /* We rely on 'ucp_wireup_search_lanes' to add fast lanes before slow
+     * lanes, so that all fast lanes are inserted to the cached ucp_ep
+     * structure */
+    if (ucp_ep_is_fast_lane(lane_type)) {
+        /* assert we don't have slow lanes until we finished with fast lanes */
+        int res = ucp_wireup_has_slow_lanes(select_ctx);
```

Comment:
"res" variable is redundant

Reply:
added it to  Avoid 'unused var' error from gcc (wrote a comment in line 750)

Reply:
use ucs_assert_always( ucp_wireup_has_slow_lanes ... ) since it's slow path code